src/client/mod.rs

```diff
@@ -37,6 +37,7 @@ use std::{
     task::{Context as FutContext, Poll},
 };
 
+use chrono::Datelike;
```

Comment:
@acdenisSK was there a push to remove chrono or are we fine with using it over time still?

Reply:
I do not recall migrating to the `time` crate over `chrono`. It should be fine to continue using it.